### PR TITLE
[CollectionCells] Add demonstration of TextCell Images

### PR DESCRIPTION
**Pull request by ianegordon**
Add a sample with a text cell with an image.

See #2911 

This issue should be fixed in #2912

### CODE REVIEW

File: components/CollectionCells/examples/CollectionCellsTextWithImageController.m [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+#import "supplemental/CollectionCellsTextWithImageController.h"
+
+static NSString *const kReusableIdentifierItem = @"itemCellIdentifier";
+static NSString *const kExampleText =
+    @"Pellentesque non quam ornare, porta urna sed, malesuada felis. Praesent at gravida felis, "
+     "non facilisis enim. Proin dapibus laoreet lorem, in viverra leo dapibus a.";
+
+@implementation CollectionCellsTextWithImageController {
+  NSMutableArray *_content;
```
Comment (yarneo) [inline components/CollectionCells/examples/CollectionCellsTextWithImageController.m]: please remove if it's not being used

File: components/CollectionCells/examples/CollectionCellsTextWithImageController.m [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  // Register cell class.
+  [self.collectionView registerClass:[MDCCollectionViewTextCell class]
+          forCellWithReuseIdentifier:kReusableIdentifierItem];
+}
+
+#pragma mark - <UICollectionViewDataSource>
+
+- (NSInteger)collectionView:(UICollectionView *)collectionView
+     numberOfItemsInSection:(NSInteger)section {
+  return 22;
```
Comment (yarneo) [inline components/CollectionCells/examples/CollectionCellsTextWithImageController.m]: maybe make this number a const in the top of the file

**Review by yarneo — COMMENTED** [review]
Is it possible to see a screenshot of how the example looks like?

**Comment by ianegordon** [comment]
screen shot 2018-02-14 at 12 58 35

**Review by yarneo — APPROVED** [review]

**Comment by ianegordon** [comment]
Kokoro seems to be consistently timing out.

**Pull request merged**